spec/cspec/AARCH64/Kernel_C.thy

```diff
@@ -6,6 +6,7 @@
 
 theory Kernel_C
 imports
+  "ASpec.Structures_A" (* FIXME AARCH64: needed for pt_index_len/vs_index_len -- maybe move these to spec/machine? *)
```

Comment:
how difficult would this be to do? I've got mixed feelings about the "CKernel depends on ASpec" change.

Reply:
I'll give this a go. The plan was to clean this up anyway, might as well do it now.